meta-ibm/meta-witherspoon/recipes-phosphor/dbus/fan-policy/mowgli/air-cooled.yaml

```diff
@@ -195,7 +195,7 @@
 
 - name: check group presence
   description: >
-    'If this condition passes more than one fan has been unplugged
+    'If this condition passes more than two fan has been unplugged
```

Comment:
This no longer matches the config below. If there 5 fan enclosures, the config reflects the previous description of `more than one` fan has been unplugged. Here, if you want `more than two fans have been unplugged`, a count of `< 3` should be used since missing 2 fans get you `= 3`, missing more than 2 fans get you `< 3`

Reply:
done